ci/htcondor.sh

```diff
@@ -3,26 +3,21 @@
 function jobqueue_before_install {
     docker version
 
-    docker run -d --name jobqueue-htcondor-mini htcondor/mini:el7 # might fail if called as script
+    docker run --rm -d --name jobqueue-htcondor-mini tillkit/dask-jobqueue-ci:htcondor #TODO: changeme on final
```

Comment:
htcondor will be changed to docker-compose too, I did not want to put it into one PR, but same as compose: imagename needs to be adjusted once an official autobuild is there